python/cudf/cudf/_lib/csv.pyx

```diff
@@ -494,29 +532,38 @@ def _get_cudf_compatible_str_from_dtype(dtype):
             "supported in CSV reader"
         )
 
-    if (
-        str(dtype) in cudf.utils.dtypes.ALL_TYPES or
-        str(dtype) in {
-            "hex", "hex32", "hex64", "date", "date32", "timestamp",
-            "timestamp[us]", "timestamp[s]", "timestamp[ms]", "timestamp[ns]",
-            "date64"
-        }
-    ):
-        return str(dtype)
-    pd_dtype = pd.core.dtypes.common.pandas_dtype(dtype)
-
-    if pd_dtype in cudf.utils.dtypes.pandas_dtypes_to_cudf_dtypes:
-        return str(cudf.utils.dtypes.pandas_dtypes_to_cudf_dtypes[pd_dtype])
-    elif isinstance(pd_dtype, np.dtype) and pd_dtype.kind in ("O", "U"):
```

Comment:
is this still supported (left side of the diff)?

Reply:
Yes, that is being taken care of in `cudf.dtype` call.